frontend/src/Settings/Indexers/Options/IndexerOptions.js

```diff
@@ -79,6 +79,19 @@ function IndexerOptions(props) {
               />
             </FormGroup>
 
+            <FormGroup>
+              <FormLabel>{translate('PreferIndexerFlags')}</FormLabel>
```

Comment:
Agreed on this one

Reply:
Yeah, just feels like a scenario where people would want to control that behaviour most of the time with CFs otherwise there will be conflicts (like we have with propers/repacks), but that option was a lot to do with not wanting to break existing behaviour and in this case it's all new.